habitat_baselines/rl/hrl/hierarchical_policy.py

```diff
@@ -53,7 +55,11 @@ def __init__(
 
             cls = eval(skill_config.skill_name)
             skill_policy = cls.from_config(
-                skill_config, observation_space, action_space, self._num_envs
+                skill_config,
+                observation_space,
+                action_space,
+                self._num_envs,
+                full_config,
```

Comment:
config, full_config, skill_config is a little bit confusing. We can name it with main node name: like policy_config. Overall if Policy need access to whole main config then it is better to pass specifics parts that needed from that main config. Otherwise we have uncontrolled access to config parameters across modules.